tests/test_e2e_10_mef_eline.py

```diff
@@ -489,23 +506,86 @@ def test_045_create_circuit_reusing_same_vlanid_from_previous_evc(self):
         h2.cmd('ip link del vlan125')
         self.net.restart_kytos_clean()
 
-    def test_050_patch_evc_new_name(self):
-        # TODO
-        assert True
+    def test_060_on_primary_path_fail_should_migrate_to_backup(self):
+        payload = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {
+                    "tag_type": 1,
+                    "value": 101
+                }
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ],
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}
+            ]
+        }
 
-    def test_055_patch_evc_new_unis(self):
-        # TODO
-        assert True
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201
 
-    def test_060_disable_evc(self):
-        # TODO
-        assert True
+        time.sleep(10)
 
-    def test_065_on_primary_path_fail_should_migrate_to_backup(self):
-        # TODO
-        assert True
+        # Check on the virtual switches directly for flows
+        s1, s2, s3 = self.net.net.get('s1', 's2', 's3')
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        flows_s3 = s3.dpctl('dump-flows')
 
-    def test_070_delete_evc_after_restart_kytos_and_no_switch_reconnected(self):
+        assert len(flows_s1.split('\r\n ')) == 3
+        assert len(flows_s2.split('\r\n ')) == 3
+        assert len(flows_s3.split('\r\n ')) == 3
+
+        # Command to up/down links to test if back-up path is taken
+        self.net.net.configLinkStatus('s1', 's2', 'down')
+
+        # Wait just a few seconds to give time to the controller receive and process the linkDown event
+        time.sleep(10)
+
+        # # Check on the virtual switches directly for flows
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        flows_s3 = s3.dpctl('dump-flows')
+
+        assert len(flows_s1.split('\r\n ')) == 3
+        assert len(flows_s2.split('\r\n ')) == 1
+        assert len(flows_s3.split('\r\n ')) == 3
```

Comment:
move this test and assertions to right after the ping result and before doing the assertions, make a call to configLinkStatus to bring it up again

Reply:
Resolved with the commit 936b43664e41092d237ea2fa397f8db8321023ee

Reply:
Resolved with the commit [https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/6687a9450f2f925ad550f0d27999392466d1eacf](commit)